internal/pkg/describe/status.go

```diff
@@ -341,6 +335,395 @@ func (a *apprunnerServiceStatus) HumanString() string {
+			},
+			[]string{color.Green.Sprint("■"), color.Yellow.Sprint("□")})
+		stringSummary := fmt.Sprintf("%d/%d Running task definition version %d (desired)",
+			data[desiredTaskDefVersion],
+			s.Service.RunningCount,
+			desiredTaskDefVersion)
+		fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+	}
+
+	// Write summary of HTTP health and container health.
+	if s.shouldShowHealthSummary() {
+		header := "Healthy"
+		if s.shouldShowHTTPHealth() {
+			healthyCount := s.healthyHTTPTasksCount()
+			bar := summaryBar(
+				[]int{
+					healthyCount,
+					(int)(s.Service.RunningCount) - healthyCount,
+				},
+				[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+			stringSummary := fmt.Sprintf("%d/%d Passes HTTP Health Checks", healthyCount, s.Service.RunningCount)
+			fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+			header = ""
+		}
+
+		if s.shouldShowContainerHealth() {
+			healthyCount, _, _ := s.containerHealthData()
+			bar := summaryBar(
+				[]int{
+					healthyCount,
+					(int)(s.Service.RunningCount) - healthyCount,
+				},
+				[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+			stringSummary := fmt.Sprintf("%d/%d Passes Container Health Checks", healthyCount, s.Service.RunningCount)
+
+			fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+
+		}
+	}
+
+	// Write summary of capacity providers.
+	if s.shouldShowCapacityProvider() {
+		header := "Capacity Provider"
+		fargate, spot, unset := s.capacityProviderData()
+		bar := summaryBar([]int{fargate, spot, unset}, []string{color.Blue.Sprint("f"), color.Magenta.Sprint("s"), color.Grey.Sprint("d")})
+		var cpSummaries []string
+		if fargate != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on Fargate", fargate, s.Service.RunningCount))
+		}
+		if spot != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on Fargate Spot", spot, s.Service.RunningCount))
+		}
+		if unset != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on default capacity provider", unset, s.Service.RunningCount))
+		}
+		fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, strings.Join(cpSummaries, ", "))
+	}
+	writer.Flush()
+}
+
+func (s *ecsServiceStatus) writeStoppedTasks(writer *tabwriter.Writer) {
+	if len(s.StoppedTasks) <= 0 {
+		return
+	}
+
+	fmt.Fprint(writer, color.Bold.Sprint("\nStopped Tasks\n\n"))
+	writer.Flush()
+
+	headers := []string{"Reason", "Task IDs"}
+	fmt.Fprintf(writer, "  %s\n", strings.Join(headers, "\t"))
+	fmt.Fprintf(writer, "  %s\n", strings.Join(underline(headers), "\t"))
+
+	reasonToTasks := make(map[string][]string)
+	for _, task := range s.StoppedTasks {
+		reasonToTasks[task.StoppedReason] = append(reasonToTasks[task.StoppedReason], task.ID)
+	}
+	for reason, ids := range reasonToTasks {
+		var shortIDs []string
+		for _, id := range ids {
+			shortIDs = append(shortIDs, shortTaskID(id))
+		}
+		fmt.Fprintf(writer, "  %s\t%s\n", reason, strings.Join(shortIDs, ","))
```

Comment:
not in this PR,  but it would be cool if we can do something like: 
https://github.com/aws/copilot-cli/blob/6b426aa1d7ee434a197190d30e4b06dd1d907803/internal/pkg/term/progress/status.go#L60

To ensure that long reason strings don't take over the screen.  
We should also limit the number of task IDs we display maybe the column can be "Sample task IDs" and display at most 5.

Reply:
addressed!